src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,21 @@
+// HomePage.tsx
+import { Hero, Navbar ,Restaurants, Dishes, IconsMeaning , WeekChef , AboutUs, Footer} from "../../components";
+import "./HomePage.scss";
+import {IconsData,ChefData} from "../../data";
+
+const HomePage = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
If the navbar will be on every page, you can simply place it once in the App.tsx file.